components/upload/UploadList.tsx

```diff
@@ -170,13 +172,21 @@ export default class UploadList extends React.Component<UploadListProps, any> {
       const linkProps =
         typeof file.linkProps === 'string' ? JSON.parse(file.linkProps) : file.linkProps;
 
-      const removeIcon = showRemoveIcon ? (
+      const removeIcon = showRemoveIcon ? customRemoveIcon ? (
+        <span title={locale.removeFile} className="anticon-delete" onClick={() => this.handleClose(file)}>
```

Comment:
useless className?

Reply:
I add this className since most of the tests are based on className ‘anticon-delete’ and 'anticon-download'.
`wrapper
      .find('.anticon-delete')
      .at(0)
      .simulate('click');
    expect(handleRemove).toHaveBeenCalledWith(fileList[0]);
`
If someone pass a custom icon, the icon wrapper may not have these classNames, which could cause failed tests.

Reply:
`anticon-delete` and `anticon-download` means **ant delete/download icon**, if set custom icon, we can write

```jsx
removeIcon = <i className="custom-remove-icon">remove</i>
```

`wrapper.find('.custom-remove-icon').simulate('click')` it's also been fine.

Reply:
Thank you. I will change its className into "custom-remove-icon"

Reply:
no, should remove className

Reply:
I will remove the className.